test/src/org/labkey/test/tests/cds/CDSGroupTest.java

```diff
@@ -58,43 +60,23 @@
-            "    colFilter=makeFilter(c(\"study_prot\", \"EQUAL\", \"q2\")), \n" +
-            "    containerFilter=NULL, \n" +
-            "    colNameOpt=\"rname\"\n" +
-            ")\n";
-    private static final String ELISPOT_Z110_REPORT_SOURCE = "library(Rlabkey)\n" +
-            "labkey.data <- labkey.selectRows(\n" +
-            "    baseUrl=labkey.url.base, \n" +
-            "    folderPath=labkey.url.path, \n" +
-            "    schemaName=\"study\", \n" +
-            "    queryName=\"ELISPOT\", \n" +
-            "    colSelect=\"SubjectId,SubjectVisit/Visit,visit_day,study_prot,assay_identifier,summary_level,antigen,antigen_type,peptide_pool,protein,protein_panel,protein_panel_protein,protein_panel_protein_peptide_pool,clade,cell_type,cell_name,vaccine_matched,specimen_type,functional_marker_name,functional_marker_type,response_call,mean_sfc,mean_sfc_neg,mean_sfc_raw,els_ifng_lab_source_key,lab_code,exp_assayid\", \n" +
-            "    colFilter=makeFilter(c(\"study_prot\", \"EQUAL\", \"z110\")), \n" +
-            "    containerFilter=NULL, \n" +
-            "    colNameOpt=\"rname\"\n" +
-            ")\n";
+    private static final String NAB_Q2_REPORT_SOURCE = "library(Rlabkey)\n" + "\n" + "# Select rows into a data frame called 'labkey.data'\n" + "\n" + "labkey.data <- labkey.selectRows(\n" + "    baseUrl=labkey.url.base, \n" + "    folderPath=labkey.url.path, \n" + "    schemaName=\"study\", \n" + "    queryName=\"NAb\", \n" + "    viewName=\"\", \n" + "    colSelect=\"SubjectId,SubjectVisit/Visit,visit_day,study_prot,assay_identifier,summary_level,specimen_type,antigen,antigen_type,virus,virus_type,virus_full_name,virus_species,virus_host_cell,virus_backbone,virus_insert_name,clade,neutralization_tier,tier_clade_virus,target_cell,initial_dilution,titer_ic50,titer_ic80,titer_ID50,titer_ID80,nab_response_ID50,nab_response_ID80,response_call,nab_lab_source_key,lab_code,exp_assayid,slope,vaccine_matched\", \n" + "    colFilter=makeFilter(c(\"study_prot\", \"EQUAL\", \"q2\")), \n" + "    containerFilter=NULL, \n" + "    colNameOpt=\"rname\"\n" + ")\n";
+    private static final String ELISPOT_Z110_REPORT_SOURCE = "library(Rlabkey)\n" + "labkey.data <- labkey.selectRows(\n" + "    baseUrl=labkey.url.base, \n" + "    folderPath=labkey.url.path, \n" + "    schemaName=\"study\", \n" + "    queryName=\"ELISPOT\", \n" + "    colSelect=\"SubjectId,SubjectVisit/Visit,visit_day,study_prot,assay_identifier,summary_level,antigen,antigen_type,peptide_pool,protein,protein_panel,protein_panel_protein,protein_panel_protein_peptide_pool,clade,cell_type,cell_name,vaccine_matched,specimen_type,functional_marker_name,functional_marker_type,response_call,mean_sfc,mean_sfc_neg,mean_sfc_raw,els_ifng_lab_source_key,lab_code,exp_assayid\", \n" + "    colFilter=makeFilter(c(\"study_prot\", \"EQUAL\", \"z110\")), \n" + "    containerFilter=NULL, \n" + "    colNameOpt=\"rname\"\n" + ")\n";
```

Comment:
Why'd you smush these R scripts into a single line? IntelliJ offers to switch it to a text block, which will retain the readability of the previous format.

Reply:
Reverted back